packages/ic-asset-certification/tests/common/mod.rs

```diff
@@ -0,0 +1,55 @@
+use ic_asset_certification::ASSET_CHUNK_SIZE;
```

Comment:
I've thought for a while that I'd like to split the tests in the asset router. Any tests that work with the public API of the crate can be moved out as "integration" tests and allow us to split them into different files while keeping a common file to share between tests. In this PR I've only added the new tests in this way, but I'd like to follow up to further refactor the other tests in this way.

The new tests focus on testing the bug fix, that the first chunk and second chunk are returned as expected and pass response verification.

Response verification was missing from the tests in this crate up until now.